build.py

```diff
@@ -502,4 +502,4 @@ def exclude_horizontal(item):
   Gen_compressed(search_paths_vertical, search_paths_horizontal).start()
 
   # This is run locally in a separate thread.
-  Gen_langfiles().start()
+  # Gen_langfiles().start()
```

Comment:
This is gross, but temporary until I can fully remove all of the old `i18n` fixtures from Blockly which will be fairly invasive.